Changes the default path whereto the web content is installed from
/usr/share/doc/cups to /usr/share/cups/webcontent because the files
of the CUPS web content are no documentation.
See CUPS STR #3578 and http://bugzilla.novell.com/show_bug.cgi?id=546023#c6.

Patch-Origin: openSUSE Leap 42.2

--- a/config-scripts/cups-directories.m4
+++ b/config-scripts/cups-directories.m4
@@ -323,11 +323,11 @@ fi
 AC_SUBST(MENUDIR)
 
 # Documentation files
-AC_ARG_WITH(docdir, [  --with-docdir           set path for documentation],docdir="$withval",docdir="")
+AC_ARG_WITH(docdir, [  --with-docdir           set path and DocumentRoot directive for web content, default=datadir/cups/webcontent],docdir="$withval",docdir="")
 
 if test x$docdir = x; then
-	CUPS_DOCROOT="$datadir/doc/cups"
-	docdir="$datadir/doc/cups"
+	CUPS_DOCROOT="$datadir/cups/webcontent"
+	docdir="$datadir/cups/webcontent"
 else
 	CUPS_DOCROOT="$docdir"
 fi

